<launch>
  <arg name="simulation" default="false"/>

  <node pkg="dr_phil_hardware" type="filter_scan.py" name="filter_scan">
    <param name="scan/source" value="/scan"/>
    <param name="scan/target" value="/scan_filtered"/>
    <param name="filter/min_range" value="0.2"/>
  </node>

  <include file="$(file dr_phil_hardware)/launch/arm_stack/moveit.launch"/>
  
  <!-- only run if we are not running a simulation -->
  <group if="$(eval arg('simulation')=='false')">
    <include file="$(file dr_phil_hardware)/launch/dr_phil_hardware_bringup.launch"/>
  </group>
</launch>
